lib/clients/AzureClient.py

```diff
@@ -46,6 +46,11 @@ def __init__(self, operation_name, configuration, directory_persistent, director
             self.last_operation(msg, 'failed')
             raise Exception(msg)
         self.instance_location = None
+
+        self.tags= {'instance_id' :self.INSTANCE_ID , 'job_name' :self.JOB_NAME}
+
+    max_block_size=100 * 1024 * 1024
```

Comment:
Why is it defined globally? Can we not define it for the class like self.tags? Or define in the BaseClient like [here](https://github.com/swativerma1/service-fabrik-backup-restore/blob/5933a702a93010defe9285310e16039a32f9b056/lib/clients/BaseClient.py#L29).

Reply:
Was earlier defined, so that can be used for other client classes. But as discussed with @sauravmndl ,done the changes